### PR TITLE
Update README with connecting to the HTTP server

### PR DESCRIPTION
**Pull request by razvand**
Add detailed instructions on running the HTTP reply application as a KVM virtual machine with networking support. The HTTP server will reply via a virtual network / bridge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,75 @@
...
+
+And, finally, run the application:
+```
+$ kraft build
+[...]
+Listening on port 8123...
+```
+
+The server listens on port `8123` but we can't access it, as the virtual machine doesn't have a (virtual) network connection to the host system and it doesn't have an IP address.
+So we have to create a connection and assign an IP address.
```
Comment (nderjung) [inline README.md]: This text should be put towards the end of the readme, it's a bit duplicate.

**Review by mogasergiu — APPROVED** [review]
Looks good to me.

Reviewed-by: Sergiu Moga <sergiu.moga@protonmail.com>

File: README.md (hunk 1 above)
Comment (razvand) [reply README.md]: My goal was to show that if you just run the application it won't provide you the HTTP server functionality: there's not network connection and no IP address. This is the reasoning behind using a virtual network and virtual bridge: to get a connection to the HTTP server application running inside a Unikraft VM.

Comment (nderjung) [reply README.md]: Yes, but running `kraft build` won't start it running though?

Comment (razvand) [reply README.md]: @nderjung , solved.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,66 @@
 # HTTP reply on Unikraft
 
-To build and run this application please use the `kraft` script:
+This application starts an HTTP server.
```
Comment (nderjung) [inline README.md]:
```diff
- This application starts an HTTP server.
+ This is a minimal example HTTP server.
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,66 @@
...
 
-    pip3 install git+https://github.com/unikraft/kraft.git
-    mkdir my-first-unikernel && cd my-first-unikernel
-    kraft up -p PLATFORM -m ARCHITECTURE httpreply
+To configure, build and run the application you need to have [kraft](https://github.com/unikraft/kraft) installed.
 
-For more information about `kraft` type ```kraft -h``` or read the
-[documentation](http://docs.unikraft.org).
+To be able to interact with the HTTP server, configure the application to run on the KVM platform:
```
Comment (nderjung) [inline README.md]:
```diff
- To be able to interact with the HTTP server, configure the application to run on the KVM platform:
+ To get started using this application, let's start by configuring the application.  For example:
```

**Review by nderjung — CHANGES_REQUESTED** [review]
All looks good, just needs two changes I think, see in-line.  Thanks!

**Review by nderjung — APPROVED** [review]
Thanks! 🐒 

Approved-by: Alexander Jung <a.jung@lancs.ac.uk>

**Pull request closed without merging**